src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+        <h5>Payload Id = 0</h5>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>=0 for payload is GO serial number.</td>
+                </tr>
+                <tr>
+                    <td>1-12</td>
+                    <td>GO serial number</td>
+                </tr>
+                <tr>
+                    <td>13</td>
+                    <td>0</td>
+                </tr>
+            </tbody>
+        </table>
+        <h5>Payload Id = 1</h5>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>=1 for payload is GO firmware version number.</td>
+                </tr>
+                <tr>
+                    <td>1-2</td>
+                    <td>GO firmware version: Product</td>
+                </tr>
+                <tr>
+                    <td>3-4</td>
+                    <td>GO firmware version: Major</td>
+                </tr>
+                <tr>
+                    <td>5-6</td>
+                    <td>GO firmware version: Minor</td>
+                </tr>
+            </tbody>
+        </table>
+        <h5>Payload Id = 2</h5>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>=2 for payload is IOX protocol version number.</td>
+                </tr>
+                <tr>
+                    <td>1-2</td>
+                    <td>GO firmware version: Major</td>
+                </tr>
+                <tr>
+                    <td>3-4</td>
+                    <td>GO firmware version: Minor</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 13 Protobuf Data</h4>
+        <p>Supported from protocol version 1.2.</p>
+        <p>This message allows a GO device to send a protobuf-encoded payload to the IOX. It supports a publish/subscribe model of vehicle status information. It is a response to GO Multi-Frame Data (0x1E) - Type 13. <a href="https://github.com/Geotab/android-external-device-example/blob/master/app/src/main/proto/iox_messaging.proto" target="_blank" rel="noreferrer">Protobuf Schema</a>.</p>
+    </div>
+);
+
+const sequenceDiagrams: ReactNode = (
+    <div className="paragraph">
+        <h3>Handshake</h3>
+        <p><img src="https://geotab.github.io/sdk/hardware/developing-an-iox/uml_handshake.png" alt=""/></p>
```

Comment:
missing content for alt attributes for these images

also, not sure if this was discussed as a team, but the images on the SDK site have a border around these images

.img-thumbnail, main img:not(.welcome) {
    padding: 0.25rem;
    background-color: #fff;
    border: 1px solid #dee2e6;
    border-radius: 0.25rem;
    max-width: 100%;
    height: auto;
}

I believe we currently have this styling on the pages.scss but not sure if we should piggyback off of that or create a custom class for images that require a border